kubectl-plugin/pkg/cmd/delete/delete.go

```diff
@@ -0,0 +1,152 @@
+package kubectlraydelete
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/ray-project/kuberay/kubectl-plugin/pkg/util"
+	"github.com/ray-project/kuberay/kubectl-plugin/pkg/util/client"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+type DeleteOptions struct {
+	configFlags  *genericclioptions.ConfigFlags
+	ioStreams    *genericiooptions.IOStreams
+	ResourceType util.ResourceType
+	ResourceName string
+	Namespace    string
+}
+
+var deleteExample = templates.Examples(`
+		# Delete RayCluster
+		kubectl ray delete raycluster sample-raycluster
+
+		# Delete RayJob
+		kubectl ray delete rayjob sample-rayjob
+
+		# Delete RayService
+		kubectl ray delete rayservice sample-rayservice
+	`)
```

Comment:
I think it's better to follow existing style for command arguments. That is, `delete (RAYCLUSTER | TYPE/NAME)`. You can use `kubectl ray session` as a reference.

Besides, could you add shell completion using this function?

https://github.com/ray-project/kuberay/blob/f69885b11c72d6fd7b11e8a80d3f7e23339c60ef/kubectl-plugin/pkg/cmd/session/session.go#L86

Reply:
+1

Reply:
`kubectl ray session` was a good reference. Thanks! I've changed it so that it follows `delete (RAYCLUSTER | TYPE/NAME)` and also has the shell complete function. PTAL thanks!